person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
```

Comment:
- [ ] As requested by the previous reviewer in this [review link](https://github.com/BilalLiaquat7/school_library/pull/1#discussion_r1350649008), the below line of code should be placed after `line 1` before the initialize constructior :+1: :sparkles: 
- code snippet
```ruby
 attr_accessor :name, :age 
  attr_reader :id 
```